### PR TITLE
WRQ-3494: Update package dependencies to fix security vulnerabilities

### PR DESCRIPTION
**Pull request by paul-beldean-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] I have run automated testing and it is passed
* [ ] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Update package dependencies to fix security vulnerabilities

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
There were NO breaking changes affecting this repo.

### Links
[//]: # (Related issues, references)
WRQ-3494

### Comments

Enact-DCO-1.0-Signed-off-by: Paul Beldean paul.beldean@lgepartner.com

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -20,37 +20,37 @@
...
-    "@wdio/static-server-service": "^7.29.1",
-    "chai": "4.3.7",
+    "@wdio/cli": "^7.33.0",
+    "@wdio/dot-reporter": "^7.33.0",
+    "@wdio/local-runner": "^7.33.0",
+    "@wdio/mocha-framework": "^7.33.0",
+    "@wdio/selenium-standalone-service": "^7.33.0",
+    "@wdio/spec-reporter": "^7.33.0",
+    "@wdio/static-server-service": "^7.33.0",
+    "chai": "4.3.10",
```
Comment (hong6316) [inline package.json:30]: Please make this ticket to unpinning `chai`.
And revert this change in this PR.

Comment (daniel-stoian-lgp) [reply package.json]: i created another ticket and reverted the change

**Review by hong6316 — APPROVED** [review]
👍
